### PR TITLE
Updated the commands

### LINKED ISSUE #2128: Refresh Meshery installation commands
**Description**
[This page](https://layer5.io/service-mesh-management/meshery/getting-started) has a number of platforms listed with platform-specific commands for installing Meshery.

**Expected Behavior**
_**ignore: 1. The commands under each platform need to be updated to reflect changes shown on https://meshery.io/#getting-started_
2. The logos in the screenshot should be an in-page link the the getting started section below. This can all be the same hyperlink.

Screen Shot 2021-08-27 at 6 59 52 AM

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by debjani1413**
Signed-off-by: debjani1413 <deydebjani916@gmail.com>

**Description**
Updated the commands under each platform
This PR fixes #2128 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit aa52e08d484ba2d4c019f81c392243327b45a679 at: https://612b99d42785a6804ee5d6ec--layer5ng.netlify.app

File: src/sections/Meshery/Meshery-platforms/index.js [hunk 1]
```diff
@@ -77,8 +76,8 @@ const supported_platforms = [
       <>
         <h2>Helm Chart</h2>
         <p>Install on Kubernetes using Helm:</p>
-        <Code codeString={dedent`helm repo add meshery https://meshery.io/charts/
-        helm install my-meshery meshery/meshery --version 2.1.2`}
+        <Code codeString={dedent`kubectl create namespace meshery
+        helm install meshery --namespace meshery install/kubernetes/helm/meshery`}
```
Comment (leecalcote) [inline src/sections/Meshery/Meshery-platforms/index.js:80]: Thank you, @debjani1413! Its turns out that the commands on meshery.io needed to be updated, too! I'm sorry for the run around here. This PR has all of the latest and correct commands. Will you please consider refreshing using this set? https://github.com/meshery/meshery.io/pull/538

**Comment by samirparhi-dev** [comment]
@debjani1413 , what @leecalcote said is right, actually we need to make these 2 places in sync. For your reference you can check the preview here :
https://deploy-preview-538--mesheryio-preview.netlify.app/?utm_source=github&utm_campaign=bot_dp

add below text as mentioned in the above preview and also the commands are not visible completely , so where ever you see the command is overflowing the box, make sure either you can increase the box size or make those command to scroll.

```
Get mesheryctl

Install mesheryctl with Bash, Brew, Scoop, or download directly.
Use mesheryctl
```
Please let me know if i can help anyway !

**Comment by Nikhil-Ladha** [comment]
@debjani1413 any updates here?

**Comment by leecalcote** [comment]
@debjani1413 thanks for this update!

@Nikhil-Ladha I'm hopeful that you might be able to discern if these changes are still needed or not.

**Pull request closed without merging**